<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Navigo by Cl√©ment Devevey</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_tab"/>
    <addaction name="actionClose_tab"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuBrowsing">
    <property name="title">
     <string>Browsing</string>
    </property>
    <widget class="QMenu" name="menuHistory">
     <property name="title">
      <string>History</string>
     </property>
     <addaction name="actionHistory"/>
    </widget>
    <addaction name="actionPrevious_page"/>
    <addaction name="actionNext_page"/>
    <addaction name="actionRefresh"/>
    <addaction name="actionStop_loading"/>
    <addaction name="actionHome"/>
    <addaction name="menuHistory"/>
   </widget>
   <widget class="QMenu" name="about">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_QT"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuBrowsing"/>
   <addaction name="about"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionPrevious_page">
   <property name="text">
    <string>Previous page</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
  </action>
  <action name="actionNext_page">
   <property name="text">
    <string>Next page</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionStop_loading">
   <property name="text">
    <string>Stop loading</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionNew_tab">
   <property name="text">
    <string>New tab</string>
   </property>
  </action>
  <action name="actionClose_tab">
   <property name="text">
    <string>Close tab</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_QT">
   <property name="text">
    <string>About QT</string>
   </property>
  </action>
  <action name="actionHome">
   <property name="text">
    <string>Home</string>
   </property>
  </action>
  <action name="actionHistory">
   <property name="text">
    <string>History</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
